src/tactic/norm_cast.lean

```diff
@@ -7,7 +7,7 @@ Normalizing casts inside expressions.
 -/
 
 import tactic.basic tactic.interactive tactic.converter.interactive
-import data.buffer.parser
+import data.buffer.parser data.num.basic
```

Comment:
The new doc guidelines call for a module doc immediately after the import, with the form
```lean
/-!
# A tactic for normalizing casts inside expressions

Description...

## Important definitions
* `tactic.norm_cast`
...
-/
```